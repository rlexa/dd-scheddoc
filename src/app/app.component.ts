import {ChangeDetectionStrategy, Component} from '@angular/core';

@Component({
  selector: 'app-root',
  template: `TODO`,
  imports: [],
  changeDetection: ChangeDetectionStrategy.OnPush,
})
export class AppComponent {}
