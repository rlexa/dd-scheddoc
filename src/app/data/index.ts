export * from './db-user.di';
