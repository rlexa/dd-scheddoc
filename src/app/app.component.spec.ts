import {MockBuilder, MockedComponentFixture, MockRender} from 'ng-mocks';
import {AppComponent} from './app.component';

describe('AppComponent', () => {
  let fixture: MockedComponentFixture<AppComponent>;

  beforeEach(() => MockBuilder(AppComponent));

  beforeEach(() => {
    fixture = MockRender(AppComponent);
    fixture.detectChanges();
  });

  it('has instance', () => expect(fixture.componentInstance).toBeTruthy());

  it('renders', () => expect(fixture).toMatchSnapshot());
});
