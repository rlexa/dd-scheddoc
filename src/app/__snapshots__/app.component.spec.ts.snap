// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent renders 1`] = `
<mock-render>
  <app-root>
    TODO
  </app-root>
</mock-render>
`;
